### ISSUE TITLE
Interface Size: "Double size" is smaller than 1.8.0

### ISSUE THREAD

**Issue by fmaclen** [OpenTTD/OpenTTD#7468]
## Version of OpenTTD
1.9.0

## Expected result
TL;DR `Double Size` interface scaling is smaller than it used to be in 1.8.0

I have a 3860x2160 monitor. Back in 1.8.0 setting the UI size to `Double Size` made it very comfortable to read/click around, in other words, it was perfect!

The latest 1.9.0 update added a separate option to scale font size independently. The `Font Size` set to `Double size` seems to match the old scaling settings but the `Interface Size` is significantly smaller, in fact, there is very little difference from `Normal` scaling.

Changing it to `Quad size` is so big that menus don't even fit on the screen.

## Actual result
This illustrates the difference between 1.8.0 and 1.9.0:
![image](https://user-images.githubusercontent.com/1434675/55579746-d5fe2d80-56cd-11e9-8bbd-3d1a9d2e6b97.png)

## Steps to reproduce
Game Options > Interface Settings > Double size

**Comment by nielsmh** [comment]
Cannot reproduce. I set Interface size to Double, take a screenshot of the landscape selection buttons on the main menu, and observe that they are exactly doubled in both dimensions. I then set Font size to double and take another screenshot, and observe that the pixel font is exactly pixel doubled.

The first screenshot, taken from a video, seems to be using a graphics mod containing higher resolution icons for the main toolbar.

It would be a help if the illustration image wasn't scaled but was 1:1 pixels from the original screenshots.

**Comment by PeterN** [comment]
I think this may be related to DPI-aware changes in 255fa6e1a8586de75b33b9598467d47344ed6c5e. Windows is doing its own scaling in 1.8.0, which is disabled in 1.9.0.

In that case, you'll find that 1x scaling is different between 1.8.0 and 1.9.0 as well.

**Comment by fmaclen** [comment]
I was about to post more screenshots when I realized that the 1.8.0 shot was done at `2560x1440` as opposed to `3840x2160` due to Windows `150% scaling`, basically what @PeterN just said.

Was that option disabled for good or is there a way to turn it back on?

**Comment by nielsmh** [comment]
Try these settings:
![image](https://user-images.githubusercontent.com/1062071/55581953-84ca5a00-571e-11e9-947d-b403b0768f4e.png)

**Comment by fmaclen** [comment]
@nielsmh yup, that did it! Thanks for the help 👍

**Comment by nielsmh** [comment]
Somewhat related feature request: #6797 
Closing this as it's technically "working as designed".
